re-import css/css-tables WPT

karlcow opened the pull request:
#### 4cb1093737ac38d49807f382a7f8d28875263664
<pre>
re-import css/css-tables WPT
<a href="https://bugs.webkit.org/show_bug.cgi?id=285971">https://bugs.webkit.org/show_bug.cgi?id=285971</a>
<a href="https://rdar.apple.com/problem/142948104">rdar://problem/142948104</a>

Reviewed by Anne van Kesteren.

Just re-import WPT tests to be in sync with WPT
Upstream commit: <a href="https://github.com/web-platform-tests/wpt/commit/e26477f970dc0b67d6cc403a3bb5646ee27d0b35">https://github.com/web-platform-tests/wpt/commit/e26477f970dc0b67d6cc403a3bb5646ee27d0b35</a>

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/baseline-empty-cell-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/baseline-empty-cell-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/baseline-empty-cell-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/caption-cyclic-percentage-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/caption-cyclic-percentage.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/caption-relative-positioning-expected.xht: Copied from LayoutTests/imported/w3c/web-platform-tests/css/css-tables/visibility-collapse-border-spacing-expected.xht.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/caption-relative-positioning.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/collapsed-border-partial-invalidation-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/collapsed-border-partial-invalidation-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/collapsed-border-partial-invalidation-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/collapsed-border-partial-invalidation-002-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/collapsed-border-partial-invalidation-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/collapsed-border-partial-invalidation-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/collapsed-border-partial-invalidation-003-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/collapsed-border-partial-invalidation-003-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/collapsed-border-partial-invalidation-003.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/collapsed-border-remove-row-group-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/collapsed-border-remove-row-group-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/collapsed-border-remove-row-group.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/colspan-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/colspan-002.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/colspan-003.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/crashtests/caption-repaint-crash.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/crashtests/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/inheritance.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/parsing/border-spacing-computed.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/percent-height-overflow-auto-in-restricted-block-size-cell.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/reference/subpixel-collapsed-borders-003-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/reference/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/remove-caption-from-anon-table-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/remove-caption-from-anon-table-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/remove-caption-from-anon-table.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/remove-colgroup-from-anon-table-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/remove-colgroup-from-anon-table.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/table-as-item-cell-percentage-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/table-as-item-cell-percentage-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/table-as-item-cell-percentage-002-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/table-as-item-cell-percentage-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/table-as-item-cell-percentage-003-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/table-as-item-cell-percentage-003.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/table-as-item-cell-percentage-004-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/table-as-item-cell-percentage-004.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/table-cell-inline-size-box-sizing-quirks-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/table-cell-inline-size-box-sizing-quirks-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/table-cell-inline-size-box-sizing-quirks.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/table-cell-overflow-explicit-height-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/table-cell-overflow-explicit-height-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/table-cell-overflow-explicit-height-002-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/table-cell-overflow-explicit-height-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/table-cell-overflow-explicit-height-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/table-cell-scroll-height-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/table-cell-scroll-height.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/table-intrinsic-size-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/table-intrinsic-size-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/table-intrinsic-size-002-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/table-intrinsic-size-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/table-intrinsic-size-003-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/table-intrinsic-size-003.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/table-intrinsic-size-004-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/table-intrinsic-size-004.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/tentative/padding-percentage-expected.xht: Copied from LayoutTests/imported/w3c/web-platform-tests/css/css-tables/visibility-collapse-border-spacing-expected.xht.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/tentative/padding-percentage.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/tentative/position-sticky-container.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/tentative/table-rows-with-zero-columns-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/tentative/table-rows-with-zero-columns.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/tentative/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/tfoot-crash-print-expected-mismatch.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/visibility-collapse-border-spacing-001-expected.xht: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-tables/visibility-collapse-border-spacing-expected.xht.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/visibility-collapse-border-spacing-001.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-tables/visibility-collapse-border-spacing.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/visibility-collapse-border-spacing-002-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/visibility-collapse-border-spacing-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/w3c-import.log:

Canonical link: <a href="https://commits.webkit.org/288972@main">https://commits.webkit.org/288972@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/fa39c6e4a16ccda97761dc83884ea028c677ac45

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/84905 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/4630 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/55/builds/39295 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/90050 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/35958 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/4720 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/12607 "Built successfully") | [  ~~🧪 wpe-wk2~~](https://ews-build.webkit.org/#/builders/34/builds/66076 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [✅ 🧪 win-tests](https://ews-build.webkit.org/#/builders/60/builds/23892 "Passed tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/87950 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/132/builds/3610 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/77161 "Passed tests") | [  ~~🧪 api-wpe~~](https://ews-build.webkit.org/#/builders/41/builds/46346 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | 
| | [  ~~🧪 ios-wk2-wpt~~](https://ews-build.webkit.org/#/builders/133/builds/3486 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/64/builds/31388 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/35031 "Built successfully") | 
| | [  ~~🧪 api-ios~~](https://ews-build.webkit.org/#/builders/13/builds/74303 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/63/builds/32196 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/91423 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/12244 "Built successfully") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/122/builds/8946 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/1/builds/74562 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/12474 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/72973 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/73686 "Passed tests") | 
| | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/18065 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/119/builds/16516 "Passed tests") | [  ~~🛠 playstation~~](https://ews-build.webkit.org/#/builders/134/builds/4279 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/13232 "Built successfully and passed tests") | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/12196 "Built successfully") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/12031 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/15525 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/13776 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

annevk commented on LayoutTests/TestExpectations:
```diff
@@ -7642,3 +7641,16 @@ imported/w3c/web-platform-tests/css/selectors/featureless-005.html [ ImageOnlyFa
 # webkit.org/b/285913 imported/w3c/web-platform-tests/web-animations/timing-model/timelines/timelines.html is extremely flakey
 imported/w3c/web-platform-tests/web-animations/timing-model/timelines/timelines.html [ Pass Failure ]
+
+#  re-import /css/css-tables
```
I think it would be better if these were in the existing list of css-tables failures. Ideally sorted.

annevk reviewed (approved):
Looks good, but I think TestExpectations should be organized a bit differently.

karlcow committed:
re-import css/css-tables WPT
https://bugs.webkit.org/show_bug.cgi?id=285971
rdar://problem/142948104

Reviewed by Anne van Kesteren.

Just re-import WPT tests to be in sync with WPT
Upstream commit: https://github.com/web-platform-tests/wpt/commit/e26477f970dc0b67d6cc403a3bb5646ee27d0b35

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/baseline-empty-cell-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/baseline-empty-cell-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/baseline-empty-cell-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/caption-cyclic-percentage-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/caption-cyclic-percentage.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/caption-relative-positioning-expected.xht: Copied from LayoutTests/imported/w3c/web-platform-tests/css/css-tables/visibility-collapse-border-spacing-expected.xht.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/caption-relative-positioning.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/collapsed-border-partial-invalidation-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/collapsed-border-partial-invalidation-001-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/collapsed-border-partial-invalidation-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/collapsed-border-partial-invalidation-002-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/collapsed-border-partial-invalidation-002-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/collapsed-border-partial-invalidation-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/collapsed-border-partial-invalidation-003-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/collapsed-border-partial-invalidation-003-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/collapsed-border-partial-invalidation-003.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/collapsed-border-remove-row-group-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/collapsed-border-remove-row-group-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/collapsed-border-remove-row-group.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/colspan-001.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/colspan-002.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/colspan-003.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/crashtests/caption-repaint-crash.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/crashtests/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/inheritance.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/parsing/border-spacing-computed.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/percent-height-overflow-auto-in-restricted-block-size-cell.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/reference/subpixel-collapsed-borders-003-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/reference/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/remove-caption-from-anon-table-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/remove-caption-from-anon-table-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/remove-caption-from-anon-table.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/remove-colgroup-from-anon-table-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/remove-colgroup-from-anon-table.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/table-as-item-cell-percentage-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/table-as-item-cell-percentage-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/table-as-item-cell-percentage-002-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/table-as-item-cell-percentage-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/table-as-item-cell-percentage-003-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/table-as-item-cell-percentage-003.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/table-as-item-cell-percentage-004-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/table-as-item-cell-percentage-004.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/table-cell-inline-size-box-sizing-quirks-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/table-cell-inline-size-box-sizing-quirks-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/table-cell-inline-size-box-sizing-quirks.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/table-cell-overflow-explicit-height-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/table-cell-overflow-explicit-height-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/table-cell-overflow-explicit-height-002-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/table-cell-overflow-explicit-height-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/table-cell-overflow-explicit-height-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/table-cell-scroll-height-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/table-cell-scroll-height.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/table-intrinsic-size-001-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/table-intrinsic-size-001.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/table-intrinsic-size-002-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/table-intrinsic-size-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/table-intrinsic-size-003-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/table-intrinsic-size-003.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/table-intrinsic-size-004-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/table-intrinsic-size-004.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/tentative/padding-percentage-expected.xht: Copied from LayoutTests/imported/w3c/web-platform-tests/css/css-tables/visibility-collapse-border-spacing-expected.xht.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/tentative/padding-percentage.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/tentative/position-sticky-container.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/tentative/table-rows-with-zero-columns-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/tentative/table-rows-with-zero-columns.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/tentative/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/tfoot-crash-print-expected-mismatch.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/visibility-collapse-border-spacing-001-expected.xht: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-tables/visibility-collapse-border-spacing-expected.xht.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/visibility-collapse-border-spacing-001.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/css/css-tables/visibility-collapse-border-spacing.html.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/visibility-collapse-border-spacing-002-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/visibility-collapse-border-spacing-002.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-tables/w3c-import.log:

Canonical link: https://commits.webkit.org/288972@main
